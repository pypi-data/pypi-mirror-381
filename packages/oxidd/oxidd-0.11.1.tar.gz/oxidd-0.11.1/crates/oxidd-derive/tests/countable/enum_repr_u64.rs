use oxidd_derive::Countable;

#[repr(u64)]
#[derive(Countable)]
enum Tag {
    X,
}

fn main() {}
