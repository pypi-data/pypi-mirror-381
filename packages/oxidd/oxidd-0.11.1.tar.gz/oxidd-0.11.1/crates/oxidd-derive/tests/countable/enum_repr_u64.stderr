error: can only derive `Countable` for `enum`s with `repr(u8)`
 --> tests/countable/enum_repr_u64.rs:3:8
  |
3 | #[repr(u64)]
  |        ^^^
